play/PlayerRules/CharacterCreation/Class-Specific Documentation/Power Suits/Chassis.txt

```diff
@@ -0,0 +1,19 @@
+		Light Chassis
+Chassis Name:	Mantis
+Chassis Power Req:	3 PU
+Standard Speed:		1m/action
+Maneuver Bonus:		2m/action
+
+Fort Bonus:		2
+Reflex Penalty:		30
+Perception Penalty:	25
+
+Disrupt DC:		75
+Pwn DC:			120
+
```

Comment:
We don't need fort bonuses specifically, but we do need some mathematical way to represent that force pushing or knocking over a power suit is harder than doing the same thing to an infantryman. Perhaps we should make a design doc for chassis, weapons and parts?

A perception penalty helps exacerbate the weakness to enemy assassins, marksmen and hunters. It also provides an avenue for social interplay. Having heavy power armor obscure your vision with bulletproof plates provides the opportunity to have more keen-eyed and rapidly-moving teammates spot targets and call in the support of the power suit.

We should probably add that, huh?